solidity/hardhat.config.cts

```diff
@@ -30,4 +31,10 @@ module.exports = {
     bail: true,
     import: 'tsx',
   },
+  warnings: {
+    // turn off all warnings for libs:
+    'fx-portal/**/*': {
```

Comment:
neat

Reply:
note that we'd have to specify the path for each lib we want to ignore for, didn't do every single lib because I feel it's safer to be selective about what we ignore